### ISSUE TITLE
Android? systemUImode

### ISSUE THREAD

**Issue by MichalNemec** [zezo357/flutter_meedu_videoplayer#41]
Whenever the widget is present, it automatically hides statusbar and bottom bar.
Is there a way to disable that?

**Comment by zezo357** [comment]
Changing screen manager edge to edge should enable and disable this feature

**Comment by MichalNemec** [comment]
Setting this still does it.

```
    screenManager: const ScreenManager(edgeToedge: false, forceLandScapeInFullscreen: false),
```

it seems its because of this code. It does not expose manual or "left untouched"
```
Future<void> setOverlays(bool visible) async {
    //await SystemChrome.setPreferredOrientations(this.orientations);
    if (visible) {
      await SystemChrome.setEnabledSystemUIMode(
          edgeToedge ? SystemUiMode.edgeToEdge : SystemUiMode.immersive, //<---- here
          overlays: overlays);
    } else {
      //print("Closed2");
      await SystemChrome.setEnabledSystemUIMode(SystemUiMode.manual,
          overlays: []);
    }
    //}
  }
```

**Comment by zezo357** [comment]
Will add option to just leave it as it is 


Right now I am working on fixing macos and iOS compilation in a new version.

**Comment by zezo357** [comment]
@MichalNemec are you using iOS or macos? 


I can add the things you need until media_kit fixes the macos and iOS problems they have.



But you can  use it only if you don't use iOS and macos😢

**Comment by MichalNemec** [comment]
yes, i have access to macbook air m1 and ios (simulator), because i didnt pay licence yet. If it helps you in development, then i can be testing subject. If you dont need it, then i can wait. Videos for us are not that important yet.

**Comment by zezo357** [comment]
Can you test add platforms branch ?


On macos or iOS

It fails in GitHub actions

**Comment by zezo357** [comment]
Yep same error, I guess I will wait for a fix from their side then.


Thank you very much ❤️

**Comment by MichalNemec** [comment]
> Yep same error, I guess I will wait for a fix from their side then.
> 
> Thank you very much ❤️

no problem, glad to help!

isnt it that this is missing on your side? https://github.com/alexmercerind/media_kit/blob/main/media_kit_libs_macos_video/macos/media_kit_libs_macos_video.podspec

**Comment by zezo357** [comment]
Yes I added it in the example, since I was testing the possibility of it being not direct dependencies that may be causing my problem

**Comment by zezo357** [comment]
can you test on `flutter_meedu_videoplayer: ^4.0.9-dev.1`

```
  final _meeduPlayerController = MeeduPlayerController(
      controlsStyle: ControlsStyle.primary,
      screenManager: ScreenManager(hideSystemOverlay: false));
```

**Comment by MichalNemec** [comment]
works as intended.

**Comment by zezo357** [comment]
fixed on `4.0.9`
